File: public/js/game/commands/userDataRequestCommand.js
function UserDataRequestCommand(){
    Command.call(this, UserDataRequestCommand.ID);
    this.data = true;
    this.execute = function(){
        Model.user = GameUser.instantiate(this.data.user);
        Model.dispatch(Events.USERDATA_READY)
        trace("Model.user assigned");
    }
}
UserDataRequestCommand.ID = 'userdata'
Command.add(UserDataRequestCommand.ID, UserDataRequestCommand)

File: src/main/java/com/hellsepontus/commands/UserDataRequestCommand.java
package com.hellsepontus.commands;

import com.hellsepontus.model.GameUser;

import java.util.Collections;
import java.util.HashMap;
import java.util.Map;

public class UserDataRequestCommand extends Command{
    public static final String ID = "userdata";

    public UserDataRequestCommand() {
        super(ID);
    }

    @Override
    public void execute() {
        GameUser user = getUser();
        send(ID, Collections.singletonMap("user", (Object)user));
    }
}
